### ISSUE TITLE
PWA - HFH name change

### ISSUE THREAD

**Issue by robertsamm** [episphere/connectApp#494]
In addition to the Connect PI change, we will also be updating the HFH site's name around the same time. They've changed their name from 'Henry Ford Health System'/HFHS to 'Henry Ford Health' /HFH (dropped the system).

To be changed on the PWA here:
1. Change in the consent + HIPAA form HTML display box (to be changed at the same time the consent/hipaa forms are changed for the Connect PI)
2. Change on the PWA list of locations where they select their IHCS

Noting other places this will also need to be changed but not on PWA:
3. Change on the public facing website list of locations and logo
4. Change in the consent + HIPAA forms updated versions (this will be bundled with the consent/hipaa forms changes for Connect PI)

Organizational doc on Box for Connect PI change: https://nih.app.box.com/file/1170318838027 
Also related to this github issue: https://github.com/episphere/connectApp/issues/491

**Comment by Davinkjohnson** [comment]
@robertsamm @depietrodeanna  It looks like there's still work to do with this? 
It appears that the name change made it into the HIPAA and Consent forms (both HTML and PDFs) 

I believe my team still needs to resolve this: 
Change on the PWA list of locations where they select their IHCS

I looked and this has not changed on either:
https://dceg.cancer.gov/research/who-we-study/cohorts/connect
https://www.cancer.gov/connect-prevention-study/who-can-join

There are some other items in the PI change doc that I'm sure need to be implemented as well (logo changes, PROTECT, and DD)

**Comment by depietrodeanna** [comment]
Thank you @Davinkjohnson. I can coordinate with the correct team to change the DCEG page and I can update the who can join page. @robertsamm we are good to make this edit now that the new consent forms are approved, yes?

**Comment by depietrodeanna** [comment]
@robertsamm yes, HFH has been swapping their logos out over time. I will update Kirsti on our plan for changing the name in the public facing website and DCEG page. I notice the logo on our cancer.gov page also needs to be updated. It's not on our [tracking doc](https://nih.box.com/s/dy9xedcd6kfxro1r1dpxgzjtpjg7sxa4), but would there be anything NORC related that needs updating? I can reach out to Stephanie and inquire if we haven't yet done that.

**Comment by robertsamm** [comment]
@depietrodeanna  Yes, please add to the list for the logo on the cancer.gov page. Do we also need to swap the logo in the stock PowerPoint slide? Aileen has already updated the map. I'm not sure if we need anything for NORC, if you don't mind asking that would be great.

**Comment by depietrodeanna** [comment]
We will need to update the PowerPoint slides in our stock deck here: [https://nih.box.com/s/dcpbvydazxv6sayyvypj0pyp9nsmcx0a](url)
I'm sure there are other slides that are outdated, but this is the one we got IRB approved way back for sites to pull from. 
I'll touch base with NORC!

**Comment by Davinkjohnson** [comment]
@robertsamm is there anything left to do on this? (Can it be closed if something like the logo swapping is being tracked elsewhere?)
